services/src/main/scala/cromwell/services/keyvalue/KeyValueServiceActor.scala

```diff
@@ -49,3 +60,22 @@ trait KeyValueServiceActor extends Actor {
     }
   }
 }
+
+/**
+  * Perhaps not great in production, but fine for test cases...!
```

Comment:
I assumed you thumbsuping a comment meant that you were going to make the change :)